libraries/botbuilder-dialogs-adaptive/tsconfig.json

```diff
@@ -8,7 +8,9 @@
     "declarationMap": true,
     "outDir": "./lib",
     "rootDir": "./src",
-    "types" : ["node"]
+    "types" : ["node"],
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
These are required for the Lodash import to work properly?

Reply:
right, TypeScript will throw error: can only be default-imported using the 'esModuleInterop' flag. 
After these two flag set to true. The error disappeared.